### DISCUSSION TITLE
[CSS/BUG] Organization's page markup problem with small screen sizes

### DISCUSSION THREAD

**Discussion by T1mL3arn** [community/community#6356] [Other Feature Feedback, Questions, & Ideas] [answered]
1. Open organization page, e.g. https://github.com/HaxeFoundation/
2. Make screen width less than 1311px (e.g. in Firefox press Ctrl+Shift+M and set the size you want)
3. Note the page is wider than screen width and has horizontal scrollbar:
![image](https://user-images.githubusercontent.com/9349164/137440239-347571a1-5a53-49de-a44e-c813331de8eb.png)

Width bigger than 1310px has no such problem:

![image](https://user-images.githubusercontent.com/9349164/137440462-47ff499a-12b3-4fc2-93bd-dbf402dedd99.png)

Is this the right place to report such issues?

**Comment by MathiasReker** [comment, marked as answer]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Reply by T1mL3arn** [reply]
Thanks for the link. The bug is still there, so I reported it with that form.

I also fixed it on my side with Stylus and simple css rule:

```css
.orghead ~ * > .gutter-condensed.gutter-lg.flex-column.flex-md-row.d-flex {
    width: 100%
}
```

**Reply by T1mL3arn** [reply]
Looks like the problem is fixed.
